Resources/public/js/views/actions/ez-treeactionview.js

```diff
@@ -22,20 +22,13 @@ YUI.add('ez-treeactionview', function (Y) {
      * @extends eZ.ButtonActionView
      */
     Y.eZ.TreeActionView = Y.Base.create('treeActionView', Y.eZ.ButtonActionView, [Y.eZ.Expandable], {
```

Comment:
looking at your patch, I'm starting to think that it does not make any sense to not deprecate this component as well. so I think, this one should also be deprecated completely and in the discovery view, it should be replaced by a raw ButtonActionView